docs/fastdds/security/access_control_plugin/access_control_plugin.rst

```diff
@@ -216,14 +219,14 @@ Allow Unauthenticated Participants
 **********************************
 
 This element is delimited by the ``<allow_unauthenticated_participants>`` XML element tag.
-It indicates whether the matching of a |DomainParticipant| with a remote |DomainParticipant| requires authentication.
+It indicates whether the matching of a DomainParticipant with a remote DomainParticipant requires authentication.
 The possible values for this element are:
 
-*  ``false``: the |DomainParticipant| shall enforce the authentication of remote |DomainParticipants| and
+*  ``false``: the DomainParticipant shall enforce the authentication of remote |DomainParticipants| and
    disallow matching those that cannot be successfully authenticated.
-*  ``true``: the |DomainParticipant| shall allow matching other |DomainParticipants| (event if the remote
-   |DomainParticipant| cannot authenticate) as long as there is not an already valid authentication with the same
-   |DomainParticipant|'s GUID.
+*  ``true``: the DomainParticipant shall allow matching other |DomainParticipants| (event if the remote
```

Comment:
Leave the first link to DomainParticipants as is, but remove the link on the second one.